SonarQube.TeamBuild.PreProcessor/Interfaces/IAnalyzerProvider.cs

```diff
@@ -7,6 +7,7 @@
 
 using SonarQube.Common;
 using SonarQube.TeamBuild.Integration;
+using SonarQube.TeamBuild.PreProcessor.Roslyn;
```

Comment:
You have a unit test failure - please see the Appveyor dashboard for details. The failing test is SonarQube.TeamBuild.PreProcessor.UnitTests.SonarWebServiceTest.GetInternalKeys